### PR TITLE
Adding close icon to player CTA.

### LINKED ISSUE #202: Update Close Icon in Player CTA

### PR DESCRIPTION
**Pull request by vazquez**
Fixes #202

### CODE REVIEW

**Comment by vazquez** [comment]
screenshot 2015-07-27 11 43 05

**Comment by vazquez** [comment]
ping @k88hudson

File: src/components/app-cta/app-cta.jsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ var AppCta = React.createClass({
           <span hidden={this.state.loading}>{this.getIntlMessage('open_app')}</span>
           <span hidden={!this.state.loading} className="open-loading">{this.getIntlMessage('searching')}</span>
         </button>
-        <button onClick={this.closeCta}>&#10005;<span className="sr-only">{this.getIntlMessage('close_message')}</span></button>
+        <button onClick={this.closeCta}><img src="./img/icon-close.svg" alt="close notice"/><span className="sr-only">{this.getIntlMessage('close_message')}</span></button>
```
Comment (k88hudson) [inline src/components/app-cta/app-cta.jsx]: you can actually remove the sr-only div now and just put the localized message in `alt`

File: static/img/icon-close.svg [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (k88hudson) [inline static/img/icon-close.svg]: if you could, it would be nice to optimize this with SVGO

**Comment by k88hudson** [comment]
@vazquez awesome, just a couple of nits

**Comment by vazquez** [comment]
SVGO optimized from 2.003 KiB to 0.632 KiB :+1:

**Comment by k88hudson** [comment]
@vazquez R+

**Pull request merged**